.eslintrc.cjs

```diff
@@ -29,8 +29,6 @@ module.exports = {
     ],
 
     "@typescript-eslint/unbound-method": "error",
-
-    "@typescript-eslint/no-non-null-assertion": "warn",
```

Comment:
I am personally not a fan of this one 😅 

if someone uses a non-null assertion I think that's generally for a good reason so having to add an eslint disabling comment for it feels very unnecessary/annoying to me 🤷

Reply:
Yeah I'm fine with this
We just need to verify on every PR to make sure it's not used unwisely